use-cases/using-vpp-and-kata.md

```diff
@@ -0,0 +1,76 @@
+# Setup to run VPP
+
+The Data Plane Development Kit (DPDK) is a set of libraries and drivers for
+fast packet processing. The Vector Packet Processing (VPP) is a platform
+extensible framework which provides out-of-the-box production quality
+switch and router functionality. VPP is a high performance packet-processing
+stack which can run on commodity CPUs. Enabling VPP with DPDK support can
+yield significant performance improvements over a Linux\*-bridge providing a
+switch with DPDK vhost-user ports.
+
+For more information about VPP visit their [wiki](https://wiki.fd.io/view/VPP).
+
+## Install and configure Kata Containers
+
+Follow  the [Kata Containers' setup instructions](https://github.com/kata-containers/documentation/wiki/Developer-Guide).
+
+In order to make use of vhost-user based interfaces, the container needs to be backed
+by huge pages.  After enabling huge-pages support on the host system, update the Kata configuration to
+enable memory preallocation and huge page support:
+
+```
+sudo sed -i -e 's/^# *\(enable_hugepages\).*=.*$/\1 = true/g' /usr/share/defaults/kata-containers/configuration.toml
```

Comment:
Can you add a `$ ` prefix here?

Reply:
You mentioned memory prealloc earlier, but havent enabled it in the config here. Is memory prealloc really required?